docs/docusaurus/docs/core/define_expectations/_examples/create_an_expectation.py

```diff
@@ -0,0 +1,56 @@
+"""
+This is an example script for creating an expectation with preset or runtime parameters.
+
+To test, run:
+pytest --docs-tests -k "doc_example_create_an_expectation" tests/integration/test_script_runner.py
+"""
+
+
+def set_up_context_for_example(context):
+    pass
+
+
+# EXAMPLE SCRIPT STARTS HERE:
+# <snippet name="docs/docusaurus/docs/core/define_expectations/_examples/create_an_expectation.py - full code example">
+import great_expectations as gx
+
+context = gx.get_context()
+# Hide this
+set_up_context_for_example(context)
+
+# All Expectations are found in the `gx.expectations` module.
+# This Expectation has all values set in advance:
+# <snippet name="docs/docusaurus/docs/core/define_expectations/_examples/create_an_expectation.py - preset expectation">
+preset_expectation = gx.expectations.ExpectColumnMaxToBeBetween(
+    column="passenger_count", min_value=1, max_value=6
+)
+# </snippet>
+
+# In this case, two Expectations are created that will be passed
+#  parameters at runtime, and unique lookups are defined for each
+#  Expectations' parameters.
+
+# <snippet name="docs/docusaurus/docs/core/define_expectations/_examples/create_an_expectation.py - dynamic expectations">
+passenger_expectation = gx.expectations.ExpectColumnMaxToBeBetween(
+    column="passenger_count",
+    min_value={"$PARAMETER": "expect_passenger_max_to_be_above"},
```

Comment:
Just floating an idea: Does this become easier if you make a constant for `expect_passenger_max_to_be_above`, etc, to show how they are linked to what is passed in below?

Reply:
As in:
```
EXPECT_PASSENGER_MAX_TO_BE_ABOVE="expect_passenger_max_to_be_above"

...

passenger_expectation = gx.expectations.ExpectColumnMaxToBeBetween(
    column="passenger_count",
    min_value={"$PARAMETER": EXPECT_PASSENGER_MAX_TO_BE_ABOVE},

...

expectation_parameters = {
    EXPECT_PASSENGER_MAX_TO_BE_ABOVE: 1
}
```
?

I'm not certain if that's easier to comprehend or not, but I'm not against making the change if you think it is.

Reply:
Maybe something like `MAX_PASSENGER_PARAMETER`, but that's the idea. I don't have a strong opinion here, I guess. If this were production code, I'd do it, but maybe that doesn't translate perfectly to documentation. Totally your call.